### DISCUSSION TITLE
DeprecationWarning

### DISCUSSION THREAD

**Discussion by mxm199** [immich-app/immich#14116] [General]
Can you tell me what the warning is and how critical it is? It seems to have appeared since version 1.120.1 and is on 1.120.2
![image](https://github.com/user-attachments/assets/c790e178-9d39-4b7e-b251-ea45b0d3e885)

**Comment by zackpollard** [comment]
This is a warning for us as the developers to fix. It's not a problem and will be taken care of in an upcoming release.
